utility/BUILD.bazel

```diff
@@ -23,6 +23,7 @@ kt_android_library(
         "//app:crashlytics_deps",
         artifact("org.jetbrains.kotlinx:kotlinx-coroutines-core"),
         artifact("androidx.appcompat:appcompat"),
+        artifact("androidx.work:work-runtime-ktx:2.4.0"),
```

Comment:
Remove the version--having it defined in WORKSPACE should be sufficient.

Reply:
Done.

Reply:
Resolving this now.